nbs/api/14_quarto.ipynb

```diff
@@ -50,7 +50,8 @@
     "from fastcore.shutil import rmtree,move,copytree\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def _pre(p,b=True): return '    ' * (len(p.parts)) + ('- ' if b else '  ')
</pre><p>This line was causing indentation problems. It is not needed anymore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>This is the way to go. No need to reinvent the wheel, I've learned about <code>IndentDumper</code> and I like it. Excellent solution.</p>

Reply:
<p>I intentionally left it there since I don't want to remove a function even if it doesn't get called. I will be waiting for maintainer input.</p>